README.md

```diff
@@ -1,22 +1,62 @@
-# Kinde ELixir SDK Starter Kit
+# Kinde Starter Kit - ELixir
+This is an Elixir template with [KindeAuth](https://kinde.com/docs/developer-tools/elixir-sdk/).
```

Comment:
Remove capitalization of L - to be consistent with: https://kinde.com/docs/developer-tools/elixir-sdk/